scripts/pin_the_slow_way.py

```diff
@@ -29,23 +29,23 @@
 
 
 pinned = {
-          'boost': 'boost 1.60.*',
+          'boost': 'boost 1.61.*',
           'bzip2': 'bzip2 1.0.*',
           'fontconfig': 'fontconfig 2.11.*',
-          'freetype': 'freetype 2.6*',
-          'hdf5': 'hdf5 1.8.17*',
+          'freetype': 'freetype 2.6.*',
+          'hdf5': 'hdf5 1.8.17',
```

Comment:
This should still have a `*` right? The fourth version number is ok to float. However, it is not always present so no `.` here.